pkg/endpoint/endpoint.go

```diff
@@ -1025,7 +1026,10 @@ func (e *Endpoint) leaveLocked(proxyWaitGroup *completion.WaitGroup, conf Delete
 // RegenerateWait should only be called when endpoint's state has successfully
 // been changed to "waiting-to-regenerate"
 func (e *Endpoint) RegenerateWait(reason string) error {
-	if !<-e.Regenerate(&regeneration.ExternalRegenerationMetadata{Reason: reason}) {
+	if !<-e.Regenerate(&regeneration.ExternalRegenerationMetadata{
+		Reason:            reason,
+		RegenerationLevel: regeneration.RegenerateWithoutDatapath, // Needs datapath load??
```

Comment:
This seems to be only used from the following API:

`PATCH /endpoint/{id} request`

Who knows what someone could shove in there, probably best to 
```suggestion
		RegenerationLevel: regeneration.RegenerateWithDatapathRewrite,
```

Reply:
Refactored with this change.